e2e/scripts/pageHelpers.js

```diff
@@ -310,12 +349,18 @@ export const loginShopper = async ({page, userCredentials}) => {
             '**/shopper/auth/v1/organizations/**/oauth2/token'
         )
         await page.getByRole('button', {name: /Sign In/i}).click()
-        await loginResponsePromise
-        expect((await loginResponsePromise).status()).toBe(303) // Login returns a 303 redirect to /callback with authCode and usid
-        await tokenResponsePromise
-        expect((await tokenResponsePromise).status()).toBe(200)
+        
+        const loginResponse = await loginResponsePromise
```

Comment:
Good work asserting on the response rather than waiting for an element to show up. We use this as a best practice because playwright behaves more stable when waiting for a response.